assets/js/googlesitekit/data/create-notifications-store.test.js

```diff
@@ -46,12 +42,16 @@ describe( 'createNotificationsStore store', () => {
 	} );
 
 	beforeEach( () => {
-		registry = createRegistry();
+		registry = createTestRegistry();
```

Comment:
This isn't be necessary but also, as a store composition utility we want to test in a bit more isolation than the average partial. Please keep this as before.